src/internal/invoker/Payload.ts

```diff
@@ -5,9 +5,32 @@ import { Source } from "../transport/RequestProcessor";
 
 export interface Contextual {
 
-    teamId: string;
+    /**
+     * ID of the Atomist Workspace
+     * @deprecated use workspaceId instead
+     */
+    teamId?: string;
+
+    /**
+     * ID of the Atomist Workspace
+     */
```

Comment:
Short TypeDoc like this can use the single line form: `/** ID of the Atomist workspace */`